src/Client/Core/PurgeResult.cs

```diff
@@ -12,15 +12,23 @@ public class PurgeResult
     /// Initializes a new instance of the <see cref="PurgeResult" /> class.
     /// </summary>
     /// <param name="count">The count of instances purged.</param>
-    public PurgeResult(int count)
+    /// <param name="isComplete">A value indicating whether the purge operation is complete. Default is null.</param>
+    public PurgeResult(int count, bool? isComplete = null)
```

Comment:
Same comment about this being a binary-breaking change. Let's create a second constructor overload that adds a non-nullable `bool isComplete` parameter.